coordinator-server/event-framework/mouseover.ts

```diff
@@ -1,15 +1,16 @@
+import { Browser } from "webdriverio";
 import { Event } from "./event";
 
 export class MouseOver extends Event {
     constructor(eventDescriptor) {
         super(eventDescriptor);
     }
 
-    async trigger(browser) {
+    async trigger(browser: Browser) {
         const driverElement = await super.getElement(browser);
         const { size: masterSize, position, identifier } = this.eventDescriptor;
 
-        const [slaveSize] = await browser.execute(
+        const slaveSize = (await browser.execute(
```

Comment:
i would revert to destructuring assignment, i find it easier to read, but this is only an opinion

Reply:
for me it was the other way around, but it's only because of lack of experience with JS :)) I'll revert it